packages/mdc-tab/foundation.js

```diff
@@ -92,11 +92,6 @@ class MDCTabFoundation extends MDCFoundation {
    * @param {!ClientRect=} previousIndicatorClientRect
    */
   activate(previousIndicatorClientRect) {
-    // Early exit
-    if (this.isActive()) {
```

Comment:
Does this have any problems if an already active tab is activated again when the tab is outside of the tab bar?

Reply:
Nope it looks fine, based on the demos page which calls `activate` on the tab directly (without a tab bar)